.github/workflows/test.yaml

```diff
@@ -81,6 +81,13 @@ jobs:
           kubectl get package -A
           echo "::endgroup::"
 
+      - name: Debug Events Output
+        if: ${{ failure() }}
+        run: |
+          echo "::group::kubectl get events"
+          kubectl get events -A
```

Comment:
```suggestion
          kubectl get events -A --sort-by='.lastTimestamp'
```

Reply:
might be helpful to have it sort events by timestamp, but that is more of a personal preference.

Reply:
Good call, I always forget the sort. More useful debug = more betterer.